### ISSUE TITLE
MacOS - Window corner border is cut off in high contrast mode

### ISSUE THREAD

**Issue by mrleemurray** [microsoft/vscode#270188]
Screenshot:

<img width="232" height="180" alt="Image" src="https://github.com/user-attachments/assets/90d2fa99-b4cf-4f52-9a45-2f679605e738" />

<img width="198" height="170" alt="Image" src="https://github.com/user-attachments/assets/736a5bb6-a341-4fc5-b323-71df8dc5946c" />

Environment Information:
- VS Code Version: 1.105.0-insider
- OS: macOS (arm64)
- Workspace: 2 folder(s) open

---
This issue was filed using the Issue Filer extension

**Comment by bpasero** [comment]
@mrleemurray is this macOS 26 possibly? I cannot reproduce:

<img width="1313" height="912" alt="Image" src="https://github.com/user-attachments/assets/3b34015b-2996-454e-84cf-90efc54d16df" />

**Comment by deepak1556** [comment]
Only relevant change in this area was https://github.com/electron/electron/pull/48376, seems like something that can repro with base electron app.

**Comment by avarayr** [comment]
> Yes, macOS 26
> 
> <img alt="Image" width="170" height="31" src="https://private-user-images.githubusercontent.com/25487940/498291720-aa44da4e-1358-4cc7-b633-47067263dde7.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTk4NTI4NTIsIm5iZiI6MTc1OTg1MjU1MiwicGF0aCI6Ii8yNTQ4Nzk0MC80OTgyOTE3MjAtYWE0NGRhNGUtMTM1OC00Y2M3LWI2MzMtNDcwNjcyNjNkZGU3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTEwMDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMDA3VDE1NTU1MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkzODczMDBhZjk0Y2I2MWY5NDEwNzQyMWFkNjY3Y2UxMjk3NDA1YjU2NDEyMDI5MDlkMmU2MjQ3NGI2YmZmYmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.yXlwTnaFo6bzj2JeA7Vlc7X19A3elA3N9uDB-WeyTCg">

Can you reproduce on stable VSCode? (Not insiders)
